scripts/load-breaches.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+const got = require("got");
+
+const AppConstants = require("../app-constants");
+const DBUtils = require("../db/utils");
+const pkg = require("../package.json");
+
+const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
+
+
+async function handleBreachesResponse(response) {
+  try {
+    const breachesJSON = JSON.parse(response.body);
+
+    for (const breach of breachesJSON) {
+      await DBUtils.createBreach(breach.Name, breach);
+    }
+  } catch (error) {
+    console.error(error);
+    // We can `process.exit()` here since it's a CLI script.
+    // eslint-disable-next-line no-process-exit
```

Comment:
Another potential tweak is to allow `console.warn()` and `console.error()` by tweaking the following rule:

```js
"no-console": ["warn", {allow: ["warn", "error"]}],
```

This reduces it to 9 **warnings**.

But I can tweak all these configs and rules in a separate PR since it's unrelated to this specific issue.

Reply:
You can ignore most of this noise. Submitted PR #109; _"Resolve some ESLint console warnings"_, which reduces everything to a much more reasonable 3 warnings:

```sh
$ npm run lint:js

> blurts-server@0.0.1 lint:js /Users/pdehaan/dev/github/mozilla/blurts-server
> eslint .


/Users/pdehaan/dev/github/mozilla/blurts-server/routes/oauth.js
  45:5  warning  Unexpected 'todo' comment  no-warning-comments

/Users/pdehaan/dev/github/mozilla/blurts-server/routes/user.js
  28:3  warning  Unexpected 'todo' comment     no-warning-comments
  29:3  warning  Unexpected console statement  no-console

✖ 3 problems (0 errors, 3 warnings)
```